### PR TITLE
Completed css-cascade. | Vera Yeh

### PR DESCRIPTION
**Pull request by VeraYeh**
image
image
image

### CODE REVIEW

File: css-cascade/specificity.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+* {
```
Comment (BrettAlbright) [inline css-cascade/specificity.css]: These should be in the opposite order, take another look at the instructions!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

File: css-cascade/specificity.css (hunk 1 above)
Comment (VeraYeh) [reply css-cascade/specificity.css]: Corrected, thanks for your review.
image

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**